config/application.rb

```diff
@@ -37,6 +38,16 @@ class Application < Rails::Application
     # -- all .rb files in that directory are automatically loaded after loading
     # the framework and any gems in your application.
 
+    # Logging.
+    config.lograge.enabled = true if ENV['LOGRAGE_ENABLED'].present?
```

Comment:
@whereismyjetpack one last point: you can move this line under 44 below and remove the `if`

Reply:
these are two different configuration options, I think we could probably ditch the `LOGRAGE_ENABLED` wasn't sure.